### ISSUE TITLE
reenter activity, the animation will not show

### ISSUE THREAD

**Issue by ddxxll2008** [tigerjj/FasterAnimationsContainer#9]
this animation is great, but when i exit an activity and enter it again, the animation will not show. so I add an destroy method to set instance = null when exiting, then the problem solved.

**Comment by MehulKK** [comment]
@ddxxll2008: you are right awesome animation but I face the same issue which you mentation as animation not showing after restart Activity. I already apply the same solution which you given but still I not solve it. can you please help me for this.

**Comment by ddxxll2008** [comment]
@MehulKK you can add this method
`public void destroy() {
		mHandler.removeCallbacksAndMessages(null);
		sInstance = null;
	}`

but there is another problem, the AsyncTask will not end when you destory fasteranimationcontainer. so sometimes there is a delay to show the animation. if you want the animation show immediately, you will stop use the asynctask.

you can see https://github.com/tigerjj/FasterAnimationsContainer/issues/10.

**Comment by MehulKK** [comment]
@ddxxll2008  👍  : thank you for your quick suggestion and its work for me.
